gpu-alloc/src/allocator.rs

```diff
@@ -122,12 +122,12 @@ where
     /// * Same `device` instance must be used for all interactions with one `GpuAllocator` instance
     ///   and memory blocks allocated from it.
     #[cfg_attr(feature = "tracing", tracing::instrument(skip(self, device)))]
-    pub unsafe fn alloc(
+    pub unsafe fn alloc<MD: MemoryDevice<M>>(
```

Comment:
Please, move trait bounds to where clause

Reply:
done in 72b96ce